package-lock.json

```diff
@@ -6,7 +6,8 @@
     "": {
       "name": "integreat-cms",
       "dependencies": {
-        "dropzone": "^6.0.0-beta.2"
+        "dropzone": "^6.0.0-beta.2",
+        "lucide": "^0.91.0"
       },
```

Comment:
Could you execute `npm install` once and add the changes to `package-lock.json` to this PR as well?